### ISSUE TITLE
[BUG] Can't decompile altered version of Yo-Kai Watch Puni Puni

### ISSUE THREAD

**Issue by SuperTavor** [iBotPeaches/Apktool#3251]
### Information
1. **Apktool Version (`apktool -version`)** - 2.8,1
2. **Operating System (Mac, Linux, Windows)** - Windows
3. **APK From? (Playstore, ROM, Other)** - Other
4. **Java Version (`java --version`)** - 8 I think

### Stacktrace/Logcat

I: Using Apktool 2.8.1 on mod.apk
I: Loading resource table...
W: Skipping unknown chunk data of size 264
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: C:\Users\yehon\AppData\Local\apktool\framework\1.apk
W: Could not decode attr value, using undecoded value instead: ns=, name=, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=, value=0x00000000
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=, value=0x00000000
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
I: Regular manifest package...
[Fatal Error] :2:52: Attribute name "n1:http:" associated with an element type "manifest" must be followed by the ' = ' character.
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Baksmaling classes2.dex...
I: Baksmaling classes3.dex...
Exception in thread "main" com.android.tools.smali.dexlib2.dexbacked.DexBackedDexFile$NotADexFile: Not a valid dex magic value: 00 00 00 00 00 00 00 00
        at com.android.tools.smali.dexlib2.util.DexUtil.verifyDexHeader(DexUtil.java:92)
        at com.android.tools.smali.dexlib2.dexbacked.DexBackedDexFile.getVersion(DexBackedDexFile.java:172)
        at com.android.tools.smali.dexlib2.dexbacked.DexBackedDexFile.<init>(DexBackedDexFile.java:96)
        at com.android.tools.smali.dexlib2.dexbacked.DexBackedDexFile.<init>(DexBackedDexFile.java:199)
        at com.android.tools.smali.dexlib2.dexbacked.ZipDexContainer$1.getDexFile(ZipDexContainer.java:180)
        at brut.androlib.src.SmaliDecoder.decode(SmaliDecoder.java:89)
        at brut.androlib.src.SmaliDecoder.decode(SmaliDecoder.java:37)
        at brut.androlib.ApkDecoder.decodeSourcesSmali(ApkDecoder.java:211)
        at brut.androlib.ApkDecoder.decode(ApkDecoder.java:121)
        at brut.apktool.Main.cmdDecode(Main.java:190)
        at brut.apktool.Main.main(Main.java:93)
### Steps to Reproduce
1. `apktool d [apk] `

### Frameworks
If this APK is from an OEM ROM (Samsung, HTC, LG). Please attach framework files
(`.apks` that live in `/system/framework` or `/system/priv-app`)

### APK
If this APK can be freely shared, please upload/attach a link to it.
https://apkadmin.com/w2wvplaekgdf/com.Level5.YWP_4.99.1_signed_mod_YasKashije_platinmods.com.apk.html

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? yes
2. If you are trying to install a modified apk, did you resign it? yes
3. Are you using the latest apktool version? yes

**Comment by iBotPeaches** [comment]
Most of these are already fixed in upcoming 2.8.2 release.

```
➜  3251 apktool d 3251.apk -s -f
Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true
I: Using Apktool v2.8.1-53-2274ed39-SNAPSHOT on 3251.apk
I: Loading resource table...
W: Skipping unknown chunk data of size 264
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Loading resource table from file: /home/ibotpeaches/.local/share/apktool/framework/1.apk
I: Decoding AndroidManifest.xml with resources...
W: Could not decode attr value, using undecoded value instead: ns=, name=, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1000000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=, value=0x00000000
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=, value=0x00000000
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x0100000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
W: Could not decode attr value, using undecoded value instead: ns=, name=http://schemas.android.com/apk/res/android, value=0x1200000c
I: Regular manifest package...
[Fatal Error] :2:52: Attribute name "n1:http:" associated with an element type "manifest" must be followed by the ' = ' character.
I: Copying raw classes.dex file...
I: Copying raw classes2.dex file...
I: Copying raw classes3.dex file...
I: Copying raw classes1.dex file...
I: Copying raw assets/audience_network.dex file...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
I: Copying META-INF/services directory
➜  3251 
```

The warnings come from putting the literal value (undecoded numeric) of a failed resource. This is either going to be resolved with some form of unknown resources or properly looking up multiple package applications.

* https://github.com/iBotPeaches/Apktool/issues/2514
* https://github.com/iBotPeaches/Apktool/issues/2960

So good news that the original issues that plagued this application from decoding are already resolved, so I'm closing this as the other remaining issues are tracked elsewhere.

**Comment by qyzhaojinxi** [comment]
hi，2.9.0 still get this problem

https://drive.google.com/file/d/1ZPVk3wRgXS__uYE0iAxoaUQQ_yb0dsl0/view?usp=sharing
